apps/aggiemap-angular/src/app/app.component.ts

```diff
@@ -1,13 +1,18 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { NotificationService } from '@tamu-gisc/common/ngx/ui/notification';
 import { Angulartics2GoogleAnalytics } from 'angulartics2';
 
 @Component({
   selector: 'tamu-gisc-aggiemap-app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
-  constructor(public analytics: Angulartics2GoogleAnalytics) {
+export class AppComponent implements OnInit {
+  constructor(public analytics: Angulartics2GoogleAnalytics, private ns: NotificationService) {
     analytics.startTracking();
   }
+
+  public ngOnInit(): void {
```

Comment:
I faintly remember you telling me long ago about having the return types defined on methods and how for something like `ngOnInit` that it's pointless to just remove it.